libs/openFrameworks/types/ofBaseTypes.cpp

```diff
@@ -48,6 +48,11 @@ ofBaseVideoPlayer::~ofBaseVideoPlayer(){
 
 }
 
+void ofBaseVideoPlayer::loadAsync(string name){
+	ofLogWarning("ofBaseVideoPlayer") << "loadAsync() not implemented";
```

Comment:
super minor thing but can you change this to "loadAsync() not implemented, loading synchronously";

otherwise it might seem that it has just failed.

i'll merge it afterwards.